fpdf/output.py

```diff
@@ -952,6 +953,31 @@ def _log_final_sections_sizes(self):
             LOGGER.debug("- %s: %s", label, _sizeof_fmt(section_size))
 
 
+def stream_content_for_image(
```

Comment:
It looks like this only applies to raster images. If so, then the name should probably reflect that.

Reply:
Agreed, I'm going to rename this function to include `_raster_` in it